src/MediaWiki.ts

```diff
@@ -296,6 +309,12 @@ class MediaWiki {
 
     return mwMetaData
   }
+
+  // Set capability properties, usied while mw.login
+  public async setCapabilities(articleId = 'MediaWiki:Sidebar', loginCookie?: string): Promise<void> {
```

Comment:
same here, this method should not exists because capabilities should not exist anymore and methods should be lazy See #1878